doc/book/helpers/head-link.md

```diff
@@ -8,10 +8,10 @@ elements for later retrieval and output in your layout script.
 The `HeadLink` helper has special methods for adding stylesheet links to its
 stack:
 
-- `appendStylesheet($href, $media, $conditionalStylesheet, $extras)`
-- `offsetSetStylesheet($index, $href, $media, $conditionalStylesheet, $extras)`
-- `prependStylesheet($href, $media, $conditionalStylesheet, $extras)`
-- `setStylesheet($href, $media, $conditionalStylesheet, $extras)`
+- `appendStylesheet($href, $media = 'screen', $conditionalStylesheet = '', $extras = array())`
```

Comment:
Please use `[]` instead of `array()`. (I can make that change on merge, however.)